### PR TITLE
Added MeshMap Brand Kits

### LINKED ISSUE #3629: [Brand] Add MeshMap brand kit
#### Current Behavior
https://layer5.io/brand has all of the project brand kits, but is missing MeshMap.

#### Desired Behavior
Create a new MeshMap section in the brand page.

#### Implementation
1. Create a new section for MeshMap
2. Add the following .zip as a download - 
[meshmap-brand-kit.zip](https://github.com/layer5io/layer5/files/10429792/meshmap-brand-kit.zip)

3. Add a few of the logos as samples.

#### Acceptance Tests
1. Users can see the new section.
2. Users can click on the intra-page nav menu to be directed to the in-page anchor for this new section.
3. Users are pushed a .zip file containing the brand-kit on click of the download button.

---
<h3> Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h3>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by ShuvamChatterjee28**
**Description**

Added MeshMap brand kits (with proper colors) in this page --> [Brand](https://layer5.io/company/brand/). Below is a working screenshot of the same.

Screenshot 2023-01-19 at 9 51 52 PM

This PR fixes #3629

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by ShuvamChatterjee28** [comment]
Hey @leecalcote, can you please review these changes?

**Comment by l5io** [comment]
🚀 Preview for commit 58ffcf93a18a1e8317e18b67859a663fd78f0c03 at: https://63ccd9904ca6a4523ac8e84a--layer5.netlify.app

File: src/sections/Company/Brand/Brand-components/meshmap.js [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import styled from "styled-components";
+import { SRLWrapper } from "simple-react-lightbox";
+import { Row, Col } from "../../../../reusecore/Layout";
+import Button from "../../../../reusecore/Button";
+import ColorBox from "../../../../components/ColorBox";
+import { FiDownloadCloud } from "@react-icons/all-files/fi/FiDownloadCloud";
+import MeshMapIcon from "../../../../assets/images/meshmap/meshmap-color-icon.png";
+import MeshMap from "../../../../assets/images/meshmap/meshmap-color-stacked.png";
+import MeshMapTitle from "../../../../assets/images/meshmap/meshmap-color-text.png";
```
Comment (Nikhil-Ladha) [inline src/sections/Company/Brand/Brand-components/meshmap.js:10]: Only use `SVG` versions of the image.
Though, keep the png files under the assets directory (what you have done already)

Comment (ShuvamChatterjee28) [reply src/sections/Company/Brand/Brand-components/meshmap.js:10]: Give me few mins, will be doing it now.

**Comment by ShuvamChatterjee28** [comment]
Hello @Nikhil-Ladha, I am done with the changes. Please feel free to review it.

**Comment by l5io** [comment]
🚀 Preview for commit 368abe4907e9629b15ec285dcaffa8b74c5854f1 at: https://63ccf960b123486b91c56e91--layer5.netlify.app

**Comment by Nikhil-Ladha** [comment]
I don't see the logo's being rendered in the preview, does it work for you locally?

**Comment by ShuvamChatterjee28** [comment]
Yes, @Nikhil-Ladha, just saw that. even in my mobile version, it is showing. Give me few mints, I am fixing it.

**Comment by ShuvamChatterjee28** [comment]
@Nikhil-Ladha, I am experiencing strange behaviour, when we are using all SVG, then there are no logos showing. I have searched the net, and it says that it can possibly be a size mismatch issue. Here is the [link](https://stackoverflow.com/questions/43606862/react-svg-not-showing#:~:text=SVG%20images%20display%20in%20React,white%20space%20around%20the%20image.).

Can we use 2 **svg** and 1 **png** images, please?

**Comment by l5io** [comment]
🚀 Preview for commit 7106be680788efc258bb031ec6bed479e85ab5a4 at: https://63cfb3fe8ef689061d20be21--layer5.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**